report-jtreg-comparator/src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/VariantComparator.java

```diff
@@ -23,7 +23,7 @@ public static void main(String[] args) throws Exception {
         ArrayList<File> buildsToCompare = new ArrayList<>();
         for (File job : options.getJobsProvider().getJobs()) {
             ArrayList<File> builds = Builds.getBuilds(job, options.isSkipFailed(), options.getNvrQuery(),
-                    options.getNumberOfBuilds(), options.isUseDefaultBuild(), options.getFormatter());
+                    options.getNumberOfBuilds(), options.isUseDefaultBuild(), options.getFormatter(), options.getConfiguration("nvr"));
```

Comment:
I guess that is why te variabel is called nvrConfig, as there is hardcoded "nvr", which I guess will change in time?

Reply:
Yes, all the hardcoded "nvr" references should be removed now.